### PR TITLE
chore: update packages to remove vulnerabilities

### PR DESCRIPTION
**Pull request by jimjenkins5**
Note: There is still one "low" vulnerability in `mocha`. This vulnerability is fixed in a later version, but `grunt-mocha-phantomjs` is not compatible with later versions of `mocha`.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
 node_js:
-  - "node" # Latest node version
-  - "lts/*" # Latest LTS version
-  - "10"
-  - "8"
-  - "8.10"
-  - "6"
+   - "node" # Latest node version
+   - "lts/*" # Latest LTS version
+   - '14'
```
Comment (yokuze) [inline .travis.yml]: I don't know if it matters, but this line and the next changed from double quotes to single quotes.

**Review by yokuze — CHANGES_REQUESTED** [review]
Just the one thing about single vs double quotes. Otherwise it looks good.

**Review by jthomerson — APPROVED** [review]

**Comment by jthomerson** [comment]
I merged this, but @yokuze feel free to comment if you see anything else that needs to be addressed in a subsequent PR.

**Comment by jthomerson** [comment]
Published in + fsaba@0.2.1

CC @jimjenkins5 @yokuze

**Pull request merged**